package.json

```diff
@@ -49,6 +50,7 @@
 		"@atlassian/aui": "^9.4.0",
 		"@octokit/auth-app": "^2.6.0",
 		"@octokit/graphql-schema": "^10.73.0",
+		"@octokit/webhooks-methods": "^2.0.0",
```

Comment:
What was this being used for?

Reply:
This is being used for validating the payload with webhook secret

Reply:
Gotcha. Is there another way we can do this? We're currently in the process of ripping Octokit out so not sure if introducing more Octokit packages is the way to go.

Reply:
This is a lightweight library with only two methods i.e. sign and verify the payload. If you still want to remove it, we can in incorporate the code in our app.

Reply:
I would imagine that the code underneath it is just using https://nodejs.org/api/crypto.html

We probably don't need it...

Reply:
There is a Pull Request that will hopefully go in tomorrow tha signs things, but would be a good spot for a verify as well. Already importing the crypto class mentioned.
https://github.com/atlassian/github-for-jira/pull/1217

Reply:
Yep @mboudreau , it is using crypto package underneath it.  

Thanks @joshkay10 for the update. I'll remove this package once your PR is in.

Reply:
If you merge main you will have new utils/encryption.ts file that it can sneak into.